{
	"return": ["Exception occurred in runner jobs.list_job: Traceback (most recent call last):\n  File \"/usr/lib/python2.7/dist-packages/salt/client/mixins.py\", line 395, in _low\n    data['return'] = self.functions[fun](*args, **kwargs)\nTypeError: list_job() got an unexpected keyword argument 'kwargs'\n"]
}